02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,188 @@
+
+Once the sample frame is selected, the next step is figuring out who to survey. In rare cases researchers may which to conduct a census and survey everyone on the sampling frame. However, the ability to implement a questionnaire at that scale is something few can do. Instead, researchers choose to *sample* individuals and use weights to estimate numbers in the population. There are a variety of different sampling methods that can be used, and more information on these can be found in Chapter \@ref(c05). This decision of which sampling method to use, impacts *sampling_error* and can be adjusted for in weighting.
+
+#### Example: Number of Pets in a Household {- #overview-design-sampdesign-ex}
+
+Let's use a simple example where a researcher is interested in knowing the average number of pets in a household. Our researcher will need to consider what the population of interest is for this. Specifically, are they interested in all of the U.S., a different country, or a more local area (e.g., city or state). Let's assume our researcher is interested in the number of pets in a U.S. household where there is at least one adult living (18 years old or older). In this case, using a sampling frame of mailing addresses would provide the least amount of coverage error. The United States Postal Service (USPS) creates a file of mailing addresses in the U.S. that covers about 95% of U.S. households ({cite Iannichone<!--Check the 95% number when adding citation, not sure if coverage has increased since then-->}). This file is called the Delivery Sequence File (DSF) and is used by many researchers and government agencies to reach the U.S. population. To sample these individuals, for simplicity, we will use a stratified simple random sample design, where we stratify by state. 
+
+Throughout this chapter we will build on this example research question to plan a survey. 
+
+### Data Collection Planning {#overview-design-dcplanning}
+
+With the sampling design decided, researchers can then make decisions on how to survey these individuals. Specifically, the *mode* used for contacting and surveying the sample, how frequently to send reminders and follow-ups, and the overall timeline of the study are three of the major data collection determinations. Traditionally, researchers have considered four main modes^[Other modes such as using mobile apps or text messaging can also be considered, but have typically smaller reach or are better for longitudinal studies (i.e., surveying the same individuals over many waves of a single study). For the purposes of this overview we will focus on these four main modes.] for conducting surveys:
+
+-  Computer Assisted Personal Interview (CAPI; also known as face-to-face or in-person interviewing)
```

Comment:
Nitpicky but pandemic taught us CAPI can be done via teleconference software, e.g. Zoom

Reply:
I think it is too new right now for it to be considered at this time.  There might be some research on this at AAPOR this year, so I'm going to hold off until I get a better gauge on how the field is going.  The biggest issue using Zoom is technology barriers that things like CAPI and CATI don't have.  So if we do include it, it would most likely be a 5th mode.